crates/sovereign/sovereign-chain-components/src/encoding/impls/convert.rs

```diff
@@ -67,5 +67,11 @@ delegate_components! {
 
         (Any, SovereignConsensusState):
             DecodeViaWasmClientState,
```

Comment:
shouldn't this be `EncodeViaWasmConsensusState`?

Reply:
We call it `Decode` here, because the component does not support the encoding process.

Reply:
sorry. I made a typo - I meant `Decode`. The point I was making between `ClientState` and `ConsensusState`.